resources/views/community/components/forum/post-comment-meta.blade.php

```diff
@@ -6,9 +6,18 @@
 ])
 
 <?php
+use App\Site\Enums\UserPreference;
 use Illuminate\Support\Carbon;
 
-$formatMetaTimestamp = function (string $rawDate): string {
+/** @var ?User $user */
+$user = auth()->user();
+$preferences = $user?->websitePrefs ?? 0;
+
+$formatMetaTimestamp = function (string $rawDate, int $preferences = 0): string {
+    if ($preferences && BitSet($preferences, UserPreference::Forum_ShowAbsoluteDates)) {
+        return $rawDate;
```

Comment:
It would probably be better to show the non-relative, but still reformatted date.

Setting on:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/885cb6ad-dc55-4097-984e-409c9639232b)

Setting off:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/9021fd19-544e-48ef-a6ea-904dc9fe6971)

If implemented, the other occurrences should also be updated. Consider using the `getNiceDate` helper function.

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/733d7183-92b0-4326-819d-381b66768105)